physics/closed_system.hpp

```diff
@@ -27,24 +27,7 @@ using quantities::MomentOfInertia;
 // handles rotational motion.
 // |InertialFrame| is an inertial frame in which the motion of the member bodies
 // is given; |SystemFrame| is a non-rotating frame with the same axes as
-// |InertialFrame| and whose origin is the centre of mass of |SystemFrame|.
-//
-// Note: It is more natural to manipulate the inertia tensor as a
-// |SymmetricBilinearForm<MomentOfInertia, Frame, Vector>| than a
-// |SymmetricBilinearForm<MomentOfInertia, Frame, Bivector>|.
-// The former has coordinates
-//   ⎛ ∑x²   ∑xy   ∑xy ⎞
-//   ⎜ ∑yx   ∑y²   ∑yz ⎟
-//   ⎝ ∑zx   ∑zy   ∑z² ⎠
-// whereas the latter (obtained from the former by |Anticommutator()| has the
-// coordinates
-//   ⎛ ∑(y² + z²)   -∑xy      -∑xy    ⎞
-//   ⎜    -∑yx   ∑(x² + z²)   -∑yz    ⎟
-//   ⎝    -∑zx      -∑zy   ∑(x² + y²) ⎠
-// It is however common practice in physics to use the bilinear form on
-// bivectors, and to call its eigenvalues the principal moments of inertia. When
-// interfacing with systems that use those, |AnticommutatorInverse()| and
-// |Anticommutator()| should be used.
```

Comment:
I would keep (most) of this comment in the private part, next to the place where `sum_of_inertia_tensors_` is declared.  It is still useful to document that we do not use internally the same representation that we expose.